packages/manager/modules/bm-server-components/src/netboot/template.html

```diff
@@ -55,12 +75,12 @@
                             class="form-control"
                             id="rescue"
                             name="rescue"
-                            data-ng-options="rescue as rescue.kernel for rescue in $ctrl.netboots.rescue"
+                            data-ng-options="rescue as rescue.kernel + ' - ' + rescue.description for rescue in $ctrl.netboots.rescue track by rescue.kernel"
                             data-ng-model="$ctrl.currentNetboot.rescue"
+                            data-ng-init="$ctrl.currentNetboot.rescue = { kernel: $ctrl.defaultRescue }"
```

Comment:
**nitpick:** Default value can be set in `$onInit` for better readability

Reply:
@anooparveti done. I've noticed that data-ng-init was not working: despite "rescue-customer" was the default choice in the dropdown, if user doesn't touch the dropdown at all and click 'Next', the API call has an null value for the bootId and the boot is not changing to "rescue-customer"  (the display was ok but the action not). So I had no other choice than doing it in the JS as you said.